tests/conftest.py

```diff
@@ -149,6 +149,67 @@ def global_execution_env():
         yield f, assets, n
 
 
+@pytest.fixture
+def initial_assets():
```

Comment:
As this fixture will be used by almost all execution tests, I'm wondering if we could find a better name (shorter and single name). Any idea?

Reply:
Maybe we could just name it `assets`?

Reply:
`assets` is already taken: it's a module name defining the responses from the backend.
So if we chose assets, we should IMO rename this module.

Reply:
we could use `dataenv` ?

Reply:
I was thinking of `data` but yes, it could be nice to use this one